### ISSUE TITLE
Exception has occurred - Access Violation

### ISSUE THREAD

**Issue by zerovolts** [HeapsIO/heaps#935]
Hello, I'm trying to follow the [Hello Hashlink](https://heaps.io/documentation/hello-hashlink.html) guide with VSCode and I'm getting the following error when I hit F5. (I'm on a modern PC running Windows 10)

![exception](https://user-images.githubusercontent.com/5137257/109777000-3a370e00-7bd1-11eb-9fa7-ca42bb7eb5bc.png)

There doesn't seem to be any other error information available to me. It appears to be a runtime issue, as it compiles with no errors. I've also tried running `haxe compile.hxml && hl hello.hl` in the console, but it just hangs for a few seconds and then terminates without anything happening.

I've copy/pasted the code from the site directly into my files to make sure it wasn't a typo, and for what it's worth, removing all of the Heaps-related code from the Main class/function prevents the exception.

This ticket (different repo) seems similar, but it doesn't seem like a solution was arrived at: https://github.com/vshaxe/hashlink-debugger/issues/36

```
> haxe --version
4.2.1+bf9ff69
```

```
> hl --version
1.11.0
```

```
> haxelib list
format: [3.5.0]
heaps: [1.9.0]
hldx: [1.10.0]
hlopenal: [1.5.0]
hlsdl: [1.10.0]
```

**Comment by Yanrishatum** [comment]
Fixed by https://github.com/HeapsIO/heaps/pull/933, either switch to git heaps, downgrade to Haxe 4.1 or wait for a hotfix release.

**Comment by zerovolts** [comment]
Thanks for the response; pulling git heaps fixed the issues. This is my first time using Haxe, so I assumed it was a mistake with my setup!

**Comment by ncannasse** [comment]
Fixed with Heaps 1.9.1
